From stern@rowland.harvard.edu  Tue Sep 11 00:49:01 2007
From: Alan Stern <stern@rowland.harvard.edu>
Date: Mon, 10 Sep 2007 11:35:14 -0400 (EDT)
Subject: USB: Get rid of annoying endpoint-release message
To: Greg KH <greg@kroah.com>
Message-ID: <Pine.LNX.4.44L0.0709101134260.2732-100000@iolanthe.rowland.org>


This patch (as990) removes an annoying debugging message.  Nobody
really cares when endpoint pseudo-devices are released.

Signed-off-by: Alan Stern <stern@rowland.harvard.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/core/endpoint.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/core/endpoint.c
+++ b/drivers/usb/core/endpoint.c
@@ -267,7 +267,6 @@ static void ep_device_release(struct dev
 {
 	struct ep_device *ep_dev = to_ep_device(dev);
 
-	dev_dbg(dev, "%s called for %s\n", __FUNCTION__, dev->bus_id);
 	endpoint_free_minor(ep_dev);
 	kfree(ep_dev);
 }
